modules/data-table/main/index.coffee

```diff
@@ -15,6 +21,203 @@ hx.userFacingText({
 fullWidthColSpan = 999 # the colspan used to make a cell display as an entire row
 collapseBreakPoint = 480
 
+columnOptionLookup = (options, name, id) ->
+  if options.columns isnt undefined and options.columns[id] isnt undefined and options.columns[id][name] isnt undefined
+    options.columns[id][name]
+  else
+    options[name]
+
+splitArray = (array, index) ->
+  left = if index is 0
+    []
+  else
+    array[0..index - 1]
```

Comment:
I agree, it does make more sense. I think there was some confusion when I was writing this as to what `..` meant vs `...` - It's cleaner to put it on single lines as well